tests/test_pyproject_pep517.py

```diff
@@ -38,6 +38,14 @@
 )
 
 
+_each_unignored_file = _file_processor.each_unignored_file
+
+
+def each_unignored_file_reversed(*args, **kwargs):
+    paths = list(_each_unignored_file(*args, **kwargs))
```

Comment:
This should be sorted too.
```suggestion
    paths = sorted(_each_unignored_file(*args, **kwargs), reverse=True)
```